src/runner/proton.rs

```diff
@@ -24,3 +24,64 @@ impl Runner for Proton {
         &self.info
     }
 }
+
+impl Proton {
+    /// Create a new Proton runner by trying multiple known installation paths
+    ///
+    /// This method attempts to find Proton in common installation locations:
+    /// - Steam (Linux): `~/.steam/steam/steamapps/common/Proton*`
+    /// - Steam (alternative): `~/.local/share/Steam/steamapps/common/Proton*`
+    /// - Flatpak Steam: `~/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/Proton*`
+    /// - System-wide installations: `/usr/share/steam/compatibilitytools.d/`
+    ///
+    /// Returns the first successfully created Proton runner, or an error if none are found.
+    ///
+    /// # Example
+    /// ```rust
+    /// use bottles_core::runner::Proton;
+    ///
+    /// match Proton::new() {
+    ///     Ok(proton) => println!("Found Proton: {}", proton.info().name()),
+    ///     Err(e) => println!("Proton not found: {}", e),
+    /// }
+    /// ```
+    pub fn new() -> Result<Self, crate::Error> {
+        let mut known_paths = Vec::new();
+
+        // Get home directory
+        if let Ok(home) = std::env::var("HOME") {
+            // Standard Steam paths
+            known_paths.extend(vec![
+                format!("{}/.steam/steam/steamapps/common/Proton 8.0", home),
+                format!("{}/.steam/steam/steamapps/common/Proton 7.0", home),
+                format!("{}/.steam/steam/steamapps/common/Proton 6.3", home),
+                format!("{}/.steam/steam/steamapps/common/Proton Experimental", home),
+                format!("{}/.local/share/Steam/steamapps/common/Proton 8.0", home),
+                format!("{}/.local/share/Steam/steamapps/common/Proton 7.0", home),
+                format!("{}/.local/share/Steam/steamapps/common/Proton 6.3", home),
+                format!("{}/.local/share/Steam/steamapps/common/Proton Experimental", home),
+                // Flatpak Steam
+                format!("{}/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/Proton 8.0", home),
+                format!("{}/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/Proton 7.0", home),
+                format!("{}/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/Proton 6.3", home),
+                format!("{}/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/Proton Experimental", home),
+            ]);
+        }
```

Comment:
This is wrong since we'll have our own runners in a bottles specific path

Reply:
I'm already in bed, feel free to change this